### DISCUSSION TITLE
Nested docs /API issue

### DISCUSSION THREAD

**Discussion by 013Meta** [rohit-gohri/redocusaurus#312] [Q&A]
Hi ,

I am using redocusaurus , works nicely with my openapi spec as a link, when trying to render within a sidebar I receive this 
<img width="1071" alt="Screenshot 2023-10-24 at 17 04 01" src="https://github.com/rohit-gohri/redocusaurus/assets/91766534/86aa029f-9fc0-4e2e-96ce-70ea64dd27e4">

Anyone have any idea whats going wrong here  ? Its a standard .md file as shown in the docs

```
---
title: Agent Endpoint
hide_table_of_contents: true
---

import ApiDocMdx from '@theme/ApiDocMdx';

<ApiDocMdx id="agent" />
````

The sidebar is like this 
```
  apiSidebar: [
    {
          type: 'doc',
          label: 'Agent Endpoint',
          id: 'agent',
    },
    {
      type: 'link',
      label: 'Agent Endpoint',
      href: '/agent',
    },
  ],
};
```

It remains on loading when trying to do so as a 'doc'

**Comment by rohit-gohri** [comment]
Are there any console errors?
